apps/prairielearn/src/components/SubmissionPanel.html.ts

```diff
@@ -223,28 +223,30 @@ export function SubmissionPanel({
               instance_question,
             })}
           </div>
-          <button
-            type="button"
-            class="btn btn-outline-dark btn-sm ms-2 me-2"
-            data-submission-id="${submission.id}"
-            data-bs-toggle="modal"
-            data-bs-target="#submissionInfoModal-${submission.id}"
-            aria-label="Submission info"
-          >
-            <i class="fa fa-info-circle fa-fw"></i>
-          </button>
-          <button
-            type="button"
-            class="expand-icon-container btn btn-outline-dark btn-sm text-nowrap ${!expanded
-              ? 'collapsed'
-              : ''}"
-            data-bs-toggle="collapse"
-            data-bs-target="#submission-${submission.id}-body"
-            aria-expanded="${expanded ? 'true' : 'false'}"
-            aria-controls="submission-${submission.id}-body"
-          >
-            <i class="fa fa-angle-up fa-fw ms-1 expand-icon"></i>
-          </button>
+          <div class="btn-group">
```

Comment:
Now using a `btn-group` for consistency with the question panel:

<img width="934" alt="Screenshot 2025-03-07 at 15 48 32" src="https://github.com/user-attachments/assets/b38b07e4-a13d-488f-a0a0-4ed17cf6da89" />

Reply:
As an example, I like the "hide" text on the button here. I agree that the info icon is clear enough to not need any text.